src/kernels/gemm.cu

```diff
@@ -1,348 +0,0 @@
-#include "cell/mod.hpp"
-#include "kernels/gemm.hpp"
-#include "types/mod.hpp"
```

Comment:
The kernel implementation can be added back later when all the issues are resolved. Otherwise, we have to maintain a buggy implementation.